Unaccountable-Magpie/WorkforceManagement
# Created The Departments Model Ticket #12

Purpose:
Creates the Departments Model. 

**How it fits in context of the project:**
It is referenced by the Controller and Views.

**Specific Feature affected:**
Departments Model
How to test (Be thorough!):

1. Checkout to  my branch mr-branch1.
2. Start BangazonWebApp.sln from the terminal.
3. Check the ERD (shown in the top left corner of the link below) to make sure the keys are appropriately. 
   represented in the model.
4. https://files.slack.com/files-pri/T03F2SDTJ-FD1NF8LKC/image_from_ios.jpg.
5. Comment out the Employee List because the Employee model has not been added yet.
6. Run the program by clicking the green arrow IISExpress button and a browser should appear with a generic webpage.
